src/pages/Defi/hooks/useFoxEthLpBalances.tsx

```diff
@@ -0,0 +1,101 @@
+import { ethAssetId } from '@shapeshiftoss/caip'
+import { DefiProvider, DefiType } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { EarnOpportunityType } from 'features/defi/helpers/normalizeOpportunity'
+import {
+  foxAssetId,
+  foxEthLpAssetId,
+  UNISWAP_V2_WETH_FOX_POOL_ADDRESS,
+} from 'features/defi/providers/fox-eth-lp/const'
+import { useFoxEthLiquidityPool } from 'features/defi/providers/fox-eth-lp/hooks/useFoxEthLiquidityPool'
+import { useLpApr } from 'plugins/foxPage/hooks/useLpApr'
+import { useEffect, useState } from 'react'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectAssetById, selectMarketDataById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+export type UseFoxEthLpBalancesReturn = {
+  opportunity: EarnOpportunityType
+  balance: string
+  foxBalance: string | null
+  ethBalance: string | null
+  loading: boolean
+}
+
+const defaultOpportunity: EarnOpportunityType = {
+  provider: DefiProvider.FoxEthLP,
+  contractAddress: UNISWAP_V2_WETH_FOX_POOL_ADDRESS,
+  rewardAddress: '',
+  tvl: '',
+  assetId: foxEthLpAssetId,
+  fiatAmount: '',
+  cryptoAmount: '',
+  chainId: 'eip155:1',
```

Comment:
replace `eip155:1` string litterals with `ethChainId` caip import in a follow-up PR

Reply:
Resolving all comments to be able to merge the PR